README for NWP Data (#1)
MeteoSwiss/opendata-forecast-data: README.md

<br>

## 2. Numerical weather forecasting model

There are two different numerical weather forecasting models, ICON-CH1-EPS and ICON-CH2-EPS, used at MeteoSwiss for the
future evolution of the athmospheric conditions in Switzerland and its surroundings. ICON-CH1-EPS provides the weather

> forecast for the next day, while ICON-CH2-EPS calculates the prognosis for the following 8 days maximum. Both models include

Reviewer: I suggest this revision for brevity and clearer distinction from the short-term forecast chapter above:
```suggestion
MeteoSwiss uses two models, **ICON-CH1-EPS** and **ICON-CH2-EPS**, to forecast atmospheric changes in Switzerland and its surroundings over a longer period than nowcasting, providing predictions for up to five days.
```

[ensemble data assimilation](https://www.meteoswiss.admin.ch/weather/warning-and-forecasting-systems/icon-forecasting-systems/ensemble-data-assimilation.html).

> The list in section [2.1](###2.2.-Parameter-metadata) shows an overview of the two different models.

Reviewer: I would remove this to shorten the text and I'd add here as a first table and category **1. Model specifications** with the models table you added below
```suggestion
```

> The user can access the forecast model output data of the last 24 hours. Data that is older than 24 hours is no longer available. To see what data is accessible, the user can search in [Catalog 1](https://sys-data.int.bgdi.ch/browser/#/collections/ch.meteoschweiz.ogd-forecasting-icon-ch1?.language=en) for data on ICON-CH1-EPS and in [Catalog 2](https://sys-data.int.bgdi.ch/browser/#/collections/ch.meteoschweiz.ogd-forecasting-icon-ch2?.language=en) for ICON-CH2-EPS.

Reviewer: I suggest moving this section to a separate category, **Accessing Forecast Data**, placing it after **1. Model Specifications** and **2. Available Parameters**.

> The structure of the data is as follows: Each file contains several parameters. What kind of parameters are accesible is stated in the [list of variables](https://meteoswiss.sharepoint.com/:x:/s/tmsAPAPN/Ee1fZIGn92NAtiaAZ3fvhvkBa0G7yuybES5VNaEm7pGqFw?e=Ffm2qX). Here are some examples:

Reviewer: This would be in a separate category **Available Parameters**. Quick correction there is one file per parameter.
```suggestion
Users can find information about available parameters, including metadata, by referring to the [list of variables](https://meteoswiss.sharepoint.com/:x:/s/tmsAPAPN/Ee1fZIGn92NAtiaAZ3fvhvkBa0G7yuybES5VNaEm7pGqFw?e=Ffm2qX). 
```

> 
> | **Parameter**     | **Standart Unit**    |**Longname**     | **SingleLevel/MultiLevel**       | **Vertical Coordinate**     | **Horizon (Lead Time)**      | **Temporral aggregation**  | **Domain**      |
> |------------------|-----------------------------|------------------|-----------------------------|------------------|-----------------------------|------------------|-----------------------------|
> | CLCH| %|Cloud area fraction in high troposphere (above ca 400hPa)| Single Level| | | | |
> |DURSUN | s | Duration of sunshine| Single Level| | | | |
> |TOT_PREC | kg $m^{-2}$| Total precipitation| Single Level| | | | |

Reviewer: In the end, I don't think giving an example will do much good here. People can get an idea by going to the link. What do you think? Might as well keep the most key information. 
```suggestion
```

### 2.1. Data granularity, update frequency, format, volume and more

Both models have different benchmarks.

> |           | **ICON-CH1-EPS** | **ICON-CH2-EPS**| **Description**|
> |-----------|------------------|-----------------|-----------------|
> | collection name | ogd-forecasting-icon-ch1 | ogd-forecasting-icon-ch2 |
> | file name | TBA | TBA |
> | format | GRIB edition 2| GRIB edition 2|
> |data granularity | every hour | every hour |
> | update frequency | every 3 hours | every 6 hours|
> | forecast period | 33 hours | 120 hours (5 days)|
> | grid type | native icosahedral | native icosahedral |
> | horizontal grid size | 1 km | 2.1 km |
> | ensemble members | 11 | 21 |
> |volume | TBA | TBA |

Reviewer: Temporarily omitted the file name and volume since the information is not yet available. They can be added later or in a separate table, as this one is already quite lengthy.

```suggestion
| **Attributes**| **ICON-CH1-EPS** | **ICON-CH2-EPS**| 
|-----------|------------------|-----------------|
| Collection Name | `ogd-forecasting-icon-ch1` | `ogd-forecasting-icon-ch2` |
| Horizontal Grid Size | 1 km | 2.1 km |
| Ensemble Members | 11 | 21 |
| Forecast Period | 33 h | 120 h |
| Grid | Native icosahedral | Native icosahedral |
| Temporal Resolution |  1 h | 1 h |
| Model Run Interval | every 3 h | every 6 h |
| Format | GRIB edition 2 | GRIB edition 2 |
```

### 2.2. Parameter metadata

The parameter metadata is part of each GRIB file.

### 2.3. Coordinate system

The ICON-CH1-EPS and ICON-CH2-EPS model uses a native icosahedral grid inherited by the original ICON model grid. In order to regrid to another grid we provide regridding operators.

### 2.4. Data visualisation

> See [jupyter-notebook examples](https://github.com/MeteoSwiss/opendata-nwp-demos).

Reviewer: As those sections are smaller we can pack them into one **Additional Data Information** and the categories below would be subsections.
